src/Tools/Md2Html/Md2Html.csproj

```diff
@@ -1,61 +1,61 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-  <ImportGroup Label="PropertySheets">
-    <Import Project="$(SolutionDir)Config\CS_SDK.props" />
-  </ImportGroup>
-  <PropertyGroup>
-    <ProjectGuid>{0893F745-CB1A-427A-8E87-CA927273039A}</ProjectGuid>
-    <OutputType>Exe</OutputType>
-    <RootNamespace>Md2Html</RootNamespace>
-    <AssemblyName>Md2Html</AssemblyName>
-    <SharedOutputPath>$(OutputPath)\Md2Html\</SharedOutputPath>
-	  <OutputPath>bin\AnyCPU\$(Configuration)\</OutputPath>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="HtmlSanitizer">
-      <Version>5.0.372</Version>
-    </PackageReference>
-    <PackageReference Include="ILMerge">
-      <Version>3.0.41</Version>
-    </PackageReference>
-    <PackageReference Include="Markdig">
-      <Version>0.22.0</Version>
-    </PackageReference>
-    <PackageReference Include="NDesk.Options">
-      <Version>0.2.1</Version>
-    </PackageReference>
-  </ItemGroup>
-  <Target Name="AfterBuildOps" AfterTargets="Build">
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)"/>
-    <!-- the ILMergePath property points to the location of ILMerge.exe console application -->
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="$(ILMergeConsolePath) $(OutputPath)Md2Html.exe ^
+<Project Sdk="Microsoft.NET.Sdk">
+    <ImportGroup Label="PropertySheets">
+        <Import Project="$(SolutionDir)Config\CS_SDK.props" />
+    </ImportGroup>
+    <PropertyGroup>
+        <ProjectGuid>{0893F745-CB1A-427A-8E87-CA927273039A}</ProjectGuid>
+        <OutputType>Exe</OutputType>
+        <RootNamespace>Md2Html</RootNamespace>
+        <AssemblyName>Md2Html</AssemblyName>
+        <SharedOutputPath>$(OutputPath)\Md2Html\</SharedOutputPath>
+        <OutputPath>bin\AnyCPU\$(Configuration)\</OutputPath>
+    </PropertyGroup>
+    <ItemGroup>
+        <Reference Include="System" />
+        <Reference Include="System.Core" />
+        <Reference Include="System.Xml.Linq" />
+        <Reference Include="System.Data.DataSetExtensions" />
+        <Reference Include="Microsoft.CSharp" />
+        <Reference Include="System.Data" />
+        <Reference Include="System.Net.Http" />
+        <Reference Include="System.Xml" />
```

Comment:
All these references could be moved under a net48 ItemGroup
Net6 does not need them

Reply:
Solved.